site/installation/docker/README.md

```diff
@@ -7,12 +7,24 @@ description: >
     Run kpt in a docker container.
 ---
 
-Use the docker image.
+Use one of the kpt docker images.
```

Comment:
Related clean up to these docs: https://github.com/GoogleContainerTools/kpt/issues/1653